server/db/utils/postgresql/schema.psql

```diff
@@ -0,0 +1,35 @@
+\c menus
+
+CREATE TABLE dessert(
+ id INT PRIMARY KEY,
```

Comment:
Fun fact: If you make this a `SERIAL PRIMARY KEY`, it will auto-increment for you on inserts. Will probably be pretty helpful when you do post requests and don't have access to what your last id was.

Reply:
Do this for all your tables!